### DISCUSSION TITLE
Want to make a contribution to this project

### DISCUSSION THREAD

**Discussion by XiudingCai** [open-mmlab/mmgeneration#451] [How-to] [answered]
If I want to contribute to this project, such as adding new translation methods, like CUT, DCLGAN, are there any requirements?

**Comment by nbei** [comment, marked as answer]
Hi @Ezreal-XD , thanks for your  interest in `MMGEN`. The only requirement is following the contribution [guidelines](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md). Happy for you to submit a Pull Request so that we will help you to polish your code. In addition, remember to write the unit test for the newly introduced models : )
